2025/build-tooling/slides.md

```diff
@@ -16,25 +16,126 @@ is: feedback
 
 ---
 
-# Slide
+# Audience
 
-Slide content
+- Developers looking to add build step to their apps
+- Developers looking to learn more about the capabilities of the build tools
+- Developers interested in how Esri's teams are leveraging modern build tools
 
 
 ---
-layout: intro
+
+# Agenda
+
+TODO: create agenda based on the presentation outline
+
+
+
+---
+
+# Benefits of using build tools
+
+- Ease of using external libraries
+- Can import/export between files
+- Live reload
+- Better performance - minification and bundle splitting
+- Can use modern JS syntax, which will get downcompiled as needed
+- Can build complete offline apps with local assets
+- TypeScript, IntelliSence
+  - TODO: Can borrow material from my previous year's TypeScript Benefits list https://github.com/maxpatiiuk/esri-dev-summit-presentations/blob/main/2024/calcite-react/index.md#typescript-
+  - show example of bugs TypeScript can easily catch?
+  - or show autocomplete for Maps SDK APIs?
+- more customizability, control, support for advanced use cases
+
+TODO: split into multiple slides or reduce. Move TypeScript out further down?
+
+
+---
+
+# Disadvantages of build tools
+
+- More complexity to set them up and get going
+  - But, build tools are getting easier and easier to use - the barrier have never been lower
```

Comment:
I think we can just talk to disadvantages at the start, and add a summary slide at the end. This way the flow is "We're going to show the benefits of build tools... using Vite. Vite is..."